### PR TITLE
feat(ui/icon): add support for simple-icons

### LINKED ISSUE #135: Support for social icons
Our current icons package `feather-icons` doesn't have "Google" icon we need for social sign in on Cloud. 
https://github.com/feathericons/feather/issues/763#issue-505003395

We need to extend our APIs to support at least "Google" icon.

### PR DESCRIPTION
**Pull request by jozefhruska**
Closes #135

### CODE REVIEW

File: packages/ui/__stories__/Icon.stories.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import { Clipboard } from 'react-feather'
 
-import { Icon } from '../src/Icon'
+import { Icon } from '../src'
+import { Google } from '@icons-pack/react-simple-icons'
```
Comment (aigoncharov) [inline packages/ui/__stories__/Icon.stories.tsx:5]: nit: I would suggest putting absolute imports first

File: packages/ui/__stories__/Icon.stories.tsx [hunk 2]
```diff
@@ -10,12 +11,24 @@ export default {
   component: Icon,
 }
 
-export const Normal = () => <Icon icon={Clipboard} ariaLabel="Icon copy to clipboard" />
+export const FeatherIconNormal = () => <Icon icon={Clipboard} ariaLabel="Icon copy to clipboard" />
```
Comment (aigoncharov) [inline packages/ui/__stories__/Icon.stories.tsx:14]: Could you remove the screenshots for the old stories?

**Review by aigoncharov — APPROVED** [review]
LGTM!

File: packages/ui/__stories__/Icon.stories.tsx (hunk 1 above)
Comment (jozefhruska) [reply packages/ui/__stories__/Icon.stories.tsx:5]: Sure, I missed this! Thanks

File: packages/ui/__stories__/Icon.stories.tsx (hunk 2 above)
Comment (jozefhruska) [reply packages/ui/__stories__/Icon.stories.tsx:14]: Sorry, thought it does that automatically.

Comment (aigoncharov) [reply packages/ui/__stories__/Icon.stories.tsx:14]: No worries. I think we should contribute that cleanup functionality to Loki at some point

**Pull request merged**